README.md

```diff
@@ -1,20 +1,122 @@
+
+### Viewing your Responsible AI Dashboard in the AzureML studio portal
+After generating the Responsible AI insights, you can view them in your associated workspace in AzureML studio, under your model registry.
+
+![01](images/01_model_registry.png)
+1. Go to your model registry in your AzureML studio workspace
+2. Click on the model you've uploaded your Responsible AI insights for
+
+![02](images/02_model_details.png)
+3. Click on the tab for `Responsible AI dashboard (preview)` under your model details page
+
+![03](images/03_responsibleaitoolbox.png)
+4. Under the `Responsible AI dashboard (preview)` tab of your model details, you will see a list of your uploaded Responsible AI insights. You can upload more than one Responsible AI dashboards for each model. Each row represents one dashboard, with information on which components were uploaded to each dashboard (i.e. explanations, counterfactuals, etc).
+
+![04](images/04_dashboard.png)
+5. At anytime viewing the dashboard, if you wish to return to the model details page, click on `Back to model details`
+<ol type="A">
+  <li>You can view the dashboard insights for each component filtered down on a global cohort you specify. Hovering over the global cohort name will show the number of datapoints and filters in that cohort as a tooltip.</li>
+  <li>Switch which global cohort you are applying to the dashboard.</li>
+  <li>Create a new global cohort based on filters you can apply in a flyout panel.</li>
+  <li>View a list of all global cohorts created and duplicate, edit or delete them.</li>
+  <li>View a list of all Responsible AI components you've uploaded to this dashboard as well as deleting components. The layout of the dashboard will reflect the order of the components in this list.</li>
+</ol>
+
+❗ **Please note:** Error Analysis, if generated, will always be at the top of the component list in your dashboard. Selecting on the nodes of the error tree or tiles of the error heatmap will automatically generate a temporary cohort that will be populated in the components below so that you can easily experiment with looking at insights for different areas of your error distribution.
+
+![05](images/05_add_dashboard.png)
+6. In between each component you can add components by clicking the blue circular button with a plus sign. This will pop up a tooltip that will give you an option of adding whichever Responsible AI component you enabled with your SDK.
+
+#### Known limitations of viewing dashboard in AzureML studio
+Due to current lack of active compute and backend storing and recomputing your Responsible AI dashboard insights in real time, the dashboard in AzureML studio is much less robust than the dashboard generated with the open source package. To generate the full dashboard in a Jupyter python notebook, please download and use our [open source Responsible AI Dashboard SDK](https://github.com/microsoft/responsible-ai-widgets). 
```

Comment:
I'm getting a parse error on "Due to current lack of active compute and backend storing and recomputing your Responsible AI dashboard insights in real time"

Reply:
I'm not sure that 'robust' is the right adjective either.

Reply:
Trimmed a few words. Also added a line onto the end noting that if they download an RAI dashboard from AzureML and run it in a Jupyter notebook, it will be fully featured there.